### PR TITLE
Fix cleanup always being executed on delete system (bsc#1189011)

### PR DESCRIPTION
**Pull request by parlt91**
## What does this PR change?

When trying to delete a minion that is not reachable the user gets proposed to delete without cleanup. However the cleanup was being executed regardless due to a parameter from the frontend not being retrievable.

## GUI diff

No difference.

- [x] **DONE**

## Documentation

- No documentation needed: **Bugfix**

- [x] **DONE**

## Test coverage

- No tests: **Bugfix**

- [x] **DONE**

## Links

Tracks https://github.com/SUSE/spacewalk/issues/15593

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/html/src/manager/systems/delete-system.tsx
Comment (cbbayburt) [inline web/html/src/manager/systems/delete-system.tsx:37]: Nitpick:

Since @Etheryte's recent improvements, you can pass proper data objects directly to `Network` methods:

```suggestion
  handleDelete = (cleanupErr: boolean) => {
    return Network.post(`/rhn/manager/api/systems/${this.props.serverId}/delete`, { nocleanup: cleanupErr })
```

**Review by cbbayburt — APPROVED** [review]
LGTM :+1:

File: web/html/src/manager/systems/delete-system.tsx
Comment (parlt91) [reply web/html/src/manager/systems/delete-system.tsx]: That's a good point. Thanks I applied the suggested changes

**Comment by parlt91** [comment]
@juliogonzalez This PR is ready to be merged. The failing `Javascript unit tests` have nothing to to with the PR and are instead addressed here: https://github.com/uyuni-project/uyuni/pull/4086 (still waiting for approval)

**Pull request merged**
